### ISSUE TITLE
Add zoom in over hover behaviour for various elements in site

### ISSUE THREAD

**Issue by navaneethsnair1** [OpenLiberty/openliberty.io#3506]
Given below are few examples. Felt that it could make the experience more dynamic and lively. 

https://github.com/OpenLiberty/openliberty.io/assets/142485870/adcb86f3-5631-475c-a957-c40b6c739dae

https://github.com/OpenLiberty/openliberty.io/assets/142485870/e044247e-f432-4aae-b7f4-c84f1f10b217

https://github.com/OpenLiberty/openliberty.io/assets/142485870/ab8c7d1d-7339-4f64-ab51-b4e44fcfaca1

**Comment by natalie-bernhard** [comment]
Consult the accessibility Slack channel for possible issues this could cause for accessibility.

**Comment by natalie-bernhard** [comment]
I like the application of this in the first two videos, but I don't think it's necessary for the News section on the homepage as there are only two cards that already take up the majority of the page
